sql/dolphinscheduler_h2.sql

```diff
@@ -315,6 +315,7 @@ CREATE TABLE t_ds_command
     id                        int(11) NOT NULL AUTO_INCREMENT,
     command_type              tinyint(4) DEFAULT NULL,
     process_definition_id     int(11) DEFAULT NULL,
```

Comment:
Can we remove this field, it seems `process_definition_id` has not been used?

Reply:
Yes, I will do it